arm-network/2017-03-01/swagger/virtualNetwork.json

```diff
@@ -680,6 +680,48 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworks/{virtualNetworkName}/usages": {
+      "get": {
+        "operationId": "VirtualNetworks_GetUsage",
```

Comment:
@EvgenyAgafonchikov Are you referring to https://github.com/Azure/azure-rest-api-specs/blob/master/arm-network/2017-03-01/swagger/virtualNetwork.json#L184 If so that returns a model `VirtualNetworkListResult` which is same as the on you have here. 

I think it should be named as list as 1. It's a pageable operation and 2. `value` type of the model is array